docs/csharp/roslyn-sdk/source-generators-overview.md

```diff
@@ -150,9 +150,9 @@ In this guide, you'll explore the creation of a source generator using the <xref
 
 ## Next steps
 
-The [Source Generators Cookbook](https://github.com/dotnet/roslyn/blob/master/docs/features/source-generators.cookbook.md) goes over some of these examples with some recommended approaches to solving them. Additionally, we have a [set of samples available on GitHub](https://github.com/dotnet/roslyn-sdk/tree/master/samples/CSharp/SourceGenerators) that you can try on your own.
+The [Source Generators Cookbook](https://github.com/dotnet/roslyn/blob/main/docs/features/source-generators.cookbook.md) goes over some of these examples with some recommended approaches to solving them. Additionally, we have a [set of samples available on GitHub](https://github.com/dotnet/roslyn-sdk/tree/main/samples/CSharp/SourceGenerators) that you can try on your own.
```

Comment:
Good catch @Youssef1313 

Ping @mijacobs   I didn't catch the incorrect branch name in the original PR because GitHub's redirection made the link work correctly. Do we have an automated tool to catch these?